app/code/Magebit/Faq1/Controller/Adminhtml/Question/Save.php

```diff
@@ -0,0 +1,148 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magebit\Faq1\Controller\Adminhtml\Question;
+
+use Exception;
+use Magebit\Faq1\Api\QuestionRepositoryInterface;
+use Magebit\Faq1\Model\QuestionFactory;
+use Magento\Backend\App\Action;
+use Magento\Backend\Helper\Js;
+use Magebit\Faq1\Model\Question;
+use Magento\Backend\Model\View\Result\Redirect;
+use Magento\Framework\App\ObjectManager;
+use Magento\Framework\App\Request\DataPersistorInterface;
+use Magento\Framework\App\ResponseInterface;
+use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\Exception\LocalizedException;
+
+/**
+ * Class that let's us save new question and edited questions
+ */
+class Save extends Action
+{
+
+    /**
+     * @var DataPersistorInterface
+     */
+    protected DataPersistorInterface $dataPersistor;
+
+    /**
+     * @var Js
+     */
+    protected Js $_backendJsHelper;
+
+    /**
+     * @var QuestionFactory
+     */
+    private $questionFactory;
+
+    /**
+     * @var QuestionRepositoryInterface
+     */
+    private $questionRepository;
+
+    /**
+     * @param Action\Context $context
+     * @param DataPersistorInterface $dataPersistor
+     * @param Js $backendJsHelper
+     * @param QuestionFactory|null $questionFactory
+     * @param QuestionRepositoryInterface|null $questionRepository
+     */
+
+    public function __construct(
+        Action\Context $context,
+        DataPersistorInterface $dataPersistor,
+        Js $backendJsHelper,
```

Comment:
You dont need "Js $backendJsHelper," class, at least i dont see you using/calling it anywhere